docs/src/theme.ts

```diff
@@ -96,6 +96,18 @@ export const theme = createTheme({
             secondary: { value: '{colors.neutral.20.value}' },
             tertiary: { value: '{colors.neutral.20.value}' },
           },
+
+          overlay: {
```

Comment:
Are these used somewhere I'm missing, or planning to use later?

Reply:
We use these in a few places, in the toggle button and pagination, and I was using it for the text selection background (the issue Tim had). We should probably included these dark mode overrides in the default theme so users can get dark mode out of the box though.